src/js/modules/infragistics.ui.editors.js

```diff
@@ -38,36 +38,168 @@
+
+				//Get
+				var nullValue = $(".selector").%%WidgetName%%("option", "nullValue");
+
+				//Set
+				$(".selector").%%WidgetName%%("option", "nullValue", null);
+				```
+			*/
 			nullValue: null,
-			/* type="string" Sets the name attribute of the value input. This input is used to sent the value to the server. In case the target element is input and it has name attribute, but the developer has set the inputName option, so this option overwrites the value input and removes the attribute from the element. */
+			/* type="string" Sets the name attribute of the value input. This input is used to sent the value to the server. In case the target element is input and it has name attribute, but the developer has set the inputName option, so this option overwrites the value input and removes the attribute from the element.
+			```
+			//Initialize
+				$(".selector").igPercentEditor({
+				  inputName : "textField"
+				});
+
+				//Get
+				var inputName = $(".selector").igPercentEditor("option", "inputName");
+
+			//Set
+			$(".selector").igPercentEditor("option", "inputName", "textField");
+			```
+			*/
 			inputName: null,
-			/* type="bool" Gets/Sets the readonly attribute. Does not allow editing. Disables all the buttons and iteracitons applied. On submit the current value is sent into the request.*/
+			/* type="bool" Gets/Sets the readonly attribute. Does not allow editing. Disables all the buttons and iteracitons applied. On submit the current value is sent into the request.
+			```
+				//Initialize
+				$(".selector").%%WidgetName%%({
+					readOnly : true
+				});
+
+				//Get
+				var readOnly = $(".selector").%%WidgetName%%("option", "readOnly");
+
+				//Set
+				$(".selector").%%WidgetName%%("option", "readOnly", true);
+			```
+			*/
 			readOnly: false,
-			/* type="bool" Gets/Sets the disabled attribute.Does not allow editing. Disables all the buttons and iteracitons applied. On submit the current value is not sent into the request*/
+			/* type="bool" Gets/Sets the disabled attribute.Does not allow editing. Disables all the buttons and iteracitons applied. On submit the current value is not sent into the request.
+			```
+				//Initialize
+				$(".selector").%%WidgetName%%({
+					disabled : false
+				});
+
+				//Get
+				var disabled = $(".selector").%%WidgetName%%("option", "disabled");
+
+				//Set
+				$(".selector").%%WidgetName%%("option", "disabled", true);
+			```
+			*/
 			disabled: false,
-			/* type="object" Gets/Sets options supported by the igValidator widget.
-				Note: Validation rules of igValidator, such as min and max value/length are applied separately triggering errors,
+			/* type="object" Gets/Sets options supported by the [igValidator](ui.igvalidator#options) widget.
+				Note: Validation rules of [igValidator](ui.igvalidator#options), such as min and max value/length are applied separately triggering errors,
 				while similar options of the editor work to prevent wrong values from being entered.
```

Comment:
"while the corresponding options of the editor prevent values violating the defined rules from being entered."